lightkurve/utils.py

```diff
@@ -321,3 +327,78 @@ def plot_image(image, ax=None, scale='linear', origin='lower',
     if show_colorbar:
         plt.colorbar(cax, ax=ax, norm=norm, label=clabel)
     return ax
+
+
+
+def query_catalog(coordinate, catalog="KIC", radius=0.5):
+        """
+        Returns an astropy table of the sources inside a radius query.
+        Current catalogs supported are KIC, EPIC and Gaia DR2.
+
+        Parameters
+        -----------
+        coordinate : astropy.coordinates.SkyCoord
+            Coordinate to query around.
+        catalog: 'KIC', 'EPIC', or 'Gaia'
+            Indicate catalog assigned for mission.
+        radius: float
+            Radius of cone search centered on the target in arcminutes.
+            Default radius is 0.5 arcmin.
+
+        Returns
+        -------
+        result : astropy.table
+            Astropy table with the following columns
+            ID : Catalog ID from catalog.
+            RAJ200: Right ascension [degrees] (KIC & EPIC)
+            DEJ2000: Declination [degrees]    (KIC & EPIC)
+            RAJ2015.5: Right ascension [degrees] (Gaia)
+            DEJ2015.5: Declination [degrees]     (Gaia)
+            pmRA: Proper motion for right ascension [mas/year]
+            pmDEC: Proper motion for declination [mas/year]
+            Kpmag: Magnitude in Kepler band [mag]
+        """
+
+        if catalog is "Gaia":
+            log.warn('Gaia RAs and Decs are at epoch 2015.5. These RA/Decs have not been corrected.')
```

Comment:
Ahh...but they have.

Reply:
I left query_catalogue to return the RA and DEC of the epoch of each catalogue without correcting for the proper motions, the correction to be done by get_sources!